core/src/main/java/tc/oc/pgm/api/integration/RequestIntegration.java

```diff
@@ -0,0 +1,12 @@
+package tc.oc.pgm.api.integration;
+
+import java.util.List;
+import net.kyori.adventure.text.Component;
+import tc.oc.pgm.api.map.MapInfo;
+
+public interface RequestIntegration {
+
+  public List<Component> getExtraMatchInfoLines(MapInfo map);
```

Comment:
What is this used for?

Reply:
My guess is sponsors "Sponsored by XXX" added at the end of the match info, shown when you log-in or when you get teleported into the map, the typical message with:
```
----- Map Name -----
Here goes the map objective in
light blue color

Extra lines, Sponsored by Player
---------------------
```